CHANGELOG.md

```diff
@@ -24,11 +30,7 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 - Add schedule toggle to DeploymentsList and remove unused next and last run [`#1924`](https://github.com/PrefectHQ/prefect-ui-library/pull/1924)
 - Enhancement: Add a deployment list component  [`#1922`](https://github.com/PrefectHQ/prefect-ui-library/pull/1922)
 
-#### [2.4.6](https://github.com/PrefectHQ/prefect-ui-library/compare/v2.4.6...2.4.6)
-
-> 8 December 2023
-
-#### [v2.4.6](https://github.com/PrefectHQ/prefect-ui-library/compare/2.4.5...v2.4.6)
+#### [2.4.6](https://github.com/PrefectHQ/prefect-ui-library/compare/2.4.5...2.4.6)
```

Comment:
these seem... not right 🤔

Reply:
The changelog generation depends on the tags present in git. Which often isn't the same locally as on main. So its easy for this to get off. I'd just ignore this for now. We need to either fix the process or remove the changelog. This is changelog is less important now that we have changes listed in github releases.